src/styles/main.scss

```diff
@@ -1,3 +1,53 @@
-body {
-  margin: 0;
+.calendar {
+  width: 707px;
+  margin: 0 auto;
```

Comment:
The calendar should be centered both horizontally and vertically. Maybe try to add display flex to parent, which is body.